doc/xsd/tester_defs.xsd

```diff
@@ -341,31 +341,33 @@
             </documentation>
```

Comment:
Summary is wrong. It was fine in accordance with its specification.

engine/tester: relax ordering in script specification